packages/subapp-web/lib/init.js

```diff
@@ -34,12 +30,32 @@ module.exports = function setup(setupContext) {
     basePath: ""
   };
 
+  let inlineRuntimeJS = "";
+  let rtJsCdnEntryPoint;
+  if (process.env.NODE_ENV === "production") {
+    rtJsCdnEntryPoint = Object.keys(cdnJsBundles).filter(ep =>
+      cdnJsBundles[ep].includes("runtime.bundle")
```

Comment:
use `startsWith`

Reply:
can't... some clap build presets results in 'js/runtime.bundle.js' in this particular variable.

Reply:
then you are not referencing the right things

Reply:
ok i changed to using `    runtimeEntryPoints = Object.keys(assets.chunksById.js).filter(ep =>
      assets.chunksById.js[ep].startsWith("runtime.bundle")
    );`